src/card.ts

```diff
@@ -1081,7 +1081,10 @@ export class FrigateCard extends LitElement {
             view: new View({
               view:
                 this._getConfig().view.camera_select === 'current'
-                  ? this._view.view
+                  ? // Always go to live view when switching to a non-Frigate camera
+                    this._cameras?.get(camera)?.camera_name
+                    ? this._view.view
+                    : 'live'
                   : (this._getConfig().view.camera_select as FrigateCardView),
```

Comment:
PS: I didn't understand (or was able to reproduce) this line, so maybe I did a mistake.